pcsx2/PINE.cpp

```diff
@@ -170,19 +170,14 @@ int PINEServer::StartSocket()
 	return 0;
 }
 
-void PINEServer::ExecuteTaskInThread()
+void PINEServer::MainLoop()
 {
 	m_end = false;
```

Comment:
Move the initalization of m_end to the class declaration (std::atomic_bool m_end{false};).